src/main/java/fr/openmc/core/features/contest/menu/TradeMenu.java

```diff
@@ -100,10 +103,14 @@ public TradeMenu(Player owner) {
             inventory.put(slot, new ItemBuilder(this, m, itemMeta -> {
                 itemMeta.lore(lore_trades);
             }).setOnClick(inventoryClickEvent -> {
+                if (!CustomItemRegistry.hasItemsAdder()) {
+                    MessagesManager.sendMessageType(player, Component.text("§cFonctionnalité bloqué. Veuillez contactez l'administration"), Prefix.CONTEST, MessageType.ERROR, true);
+                    return;
+                }
                 String m1 = String.valueOf(inventoryClickEvent.getCurrentItem().getType());
                 int amount = (int) trade.get("amount");
                 int amount_shell = (int) trade.get("amount_shell");
-                ItemStack shell_contestItem = CustomStack.getInstance("contest:contest_shell").getItemStack();
+                ItemStack shell_contestItem = CustomStack.getInstance(namespaceShellContest).getItemStack();
```

Comment:
Ce nom de variable ne respecte pas les règles de contribution.
https://github.com/ServerOpenMC/PluginV2/blob/master/CONTRIBUTING.md#-style-de-code
```suggestion
                ItemStack shellContestItem = CustomStack.getInstance(namespaceShellContest).getItemStack();
```